### ISSUE TITLE
Handling of "clear_width"

### ISSUE THREAD

**Issue by cymed** [qgep/QGEP#830]
**Describe the bug**
In VSA DSS 2015 and VSA DSS 2020, the "clear width" (Lichte Breite) is not attributed directly, but as a ratio in the pipe profile. VSA DSS mini, on the other hand, has an attribute "Lichte_Breite", whilst the height_width_ratio is omitted. The heigth/width ratio is set as a decimal number ranging from 0.01 to 100.00. Depending on the ratio, intercompatibility is limited.

Assuming a rectangular reach with a clear height of 1'800 mm and a clear width of 1'400 mm. When receiving these values in a VSA DSS mini, VSA DSS converts it into a pipe profile with a height/width ratio of 1.29. The exact ratio would be 1.2857xxx though. When this dataset is exported into VSA DSS mini again, the clear width is set to 1'406 mm. 
Also, the limitations of VSA DSS mean that reach of 1'800 mm height and 1'400 mm width cannot be created in TEKSI Wastewater. 

**To Reproduce**
Exact steps to reproduce the behavior:
1. Create a reach with a height of 1'800 mm
2. try to set the width to 1'400 mm

**Expected behavior**
 A reach of 1'800 mm height and 1'400 mm width can be created in TEKSI Wastewater.

**Screenshots / data**
![grafik](https://github.com/QGEP/QGEP/assets/102035587/33fc8a3e-74d3-4660-ba35-5f37a59f9f39)

**Desktop (please complete the following information):**
irrelevant

**Additional context**
Possible workaround: Add the attribute "clear_width" to qgep_od.reach. For non-circular pipe profile, altering the clear_width updates the height_width_ratio in pipe_profile. Caveat: Any alteration of a clear width changes the ratio for all reaches with the same profile.

**Comment by cymed** [comment]
Yes it is. Altering the Verhaeltnis_H_B requires the following changes:

- Alter qgep_od.pipe_profile
- Alter import scripts from VSA DSS mini
- Alter export scripts to VSA DSS

**Comment by sjib** [comment]
Added to issue list for VSA Change Management (VSAID1057)

**Comment by cymed** [comment]
@sjib what is the status of VSAID 1057? It is nor listed in the Gelbdruck nor in the Blaudruck
